firmware/boron-ins-fsm/test/mocks/mock_thread.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Mock implementation for Thread functions and classes
+ */
+
+#pragma once
+
+struct os_thread_fn_t {};
+
+class Thread
+{
+public:
+    Thread() {}
+
+public:
```

Comment:
Does the `public:` need to be repeated for each public method? I thought that everything defined under it until the next public/private keyword would all be considered the same.

Reply:
Good point, there only needs to be one public keyword. I changed this in my latest commit